src/toil/common.py

```diff
@@ -1257,33 +1257,26 @@ def cacheDirName(workflowID):
 
 def getDirSizeRecursively(dirPath):
     """
-    This method will walk through a directory and return the cumulative filesize in bytes of all
-    the files in the directory and its subdirectories.
+    This method will return the cumulative filesize in bytes of all of the files
```

Comment:
This is ambiguous: "file size" usually means the number of bytes in each file. Be sure to explain that this is the number of bytes occupied by the files on disk.